app/models/submission/test_run.rb

```diff
@@ -5,6 +5,8 @@ class Submission::TestRun < ApplicationRecord
 
   scope :ops_successful, -> { where(ops_status: 200) }
 
+  delegate :solution, to: :submission
```

Comment:
@kntsoriano This should be an association not a delegate (I'll update, but just be aware of these as they cause n+1s otherwise)